parsec/mca/termdet/fourcounter/termdet_fourcounter_module.c

```diff
@@ -162,8 +162,8 @@ int parsec_termdet_fourcounter_msg_dispatch(parsec_comm_engine_t *ce, long unsig
         if ((NULL == tp) || (NULL == tp->tdm.monitor) ||
             (((parsec_termdet_fourcounter_monitor_t *) tp->tdm.monitor)->state ==
              PARSEC_TERMDET_FOURCOUNTER_NOT_READY)) {
-            delayed_msg = (parsec_termdet_fourcounter_delayed_msg_t *) malloc(
-                    sizeof(parsec_termdet_fourcounter_delayed_msg_t));
+            delayed_msg = (parsec_termdet_fourcounter_delayed_msg_t *) calloc(
+                    sizeof(parsec_termdet_fourcounter_delayed_msg_t), 1);
```

Comment:
The number of members comes first in `calloc`:

```suggestion
            delayed_msg = (parsec_termdet_fourcounter_delayed_msg_t *) calloc(
                    1, sizeof(parsec_termdet_fourcounter_delayed_msg_t));
```